hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2286,12 +2286,20 @@
       ozone.metadata.dirs.
     </description>
   </property>
+  <property>
+    <name>ozone.scm.network.topology.schema.file.type</name>
+    <value>xml</value>
+    <tag>OZONE, MANAGEMENT</tag>
```

Comment:
I think here in tag do we need to add SCM also? (As this is related to SCM)

Reply:
Thanks for review @bharatviswa504, updated the patch.

Reply:
I think we should add a new tag called network since Management sounds too wide and catch all. Or if you want to be more specific, Topology might be good too.